docs-devsite/vertexai-preview.chatsession.md

```diff
@@ -36,7 +36,7 @@ export declare class ChatSession
 
 |  Method | Modifiers | Description |
 |  --- | --- | --- |
-|  [getHistory()](./vertexai-preview.chatsession.md#chatsessiongethistory) |  | Gets the chat history so far. Blocked prompts are not added to history. Blocked candidates are not added to history, nor are the prompts that generated them. |
+|  [getHistory()](./vertexai-preview.chatsession.md#chatsessiongethistory) |  | Gets the chat history so far. Blocked prompts are not added to history. Blocked candidates are neither added to history nor are the prompts that generated them. |
```

Comment:
This is another poor usage IMO.  I'd stick with just "not," here and below.

If we insist on using neither, the correct way would be "Neither blocked candidates nor the prompts that generated them are . . ."